### PR TITLE
Default port updates for v1.6.0

### PR DESCRIPTION
**Pull request by mDuo13**
Documents https://github.com/ripple/rippled/pull/3235

### CODE REVIEW

File: content/tutorials/manage-the-rippled-server/configure-peering/cluster-rippled-servers.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ If you run multiple [`rippled` servers](the-rippled-server.html) in the same dat
 
         This defines specific peer servers to which this server should always attempt to maintain a direct peer-to-peer connection.
 
+        **Note:** If you omit the port number, the server uses port 2459, the IANA-assigned port for the XRP Ledger protocol. [New in: rippled 1.6.0][]
```
Comment (ryangyoung) [inline content/tutorials/manage-the-rippled-server/configure-peering/cluster-rippled-servers.md]: Should we add a link back to the peer protocol page from this note?

File: content/tutorials/manage-the-rippled-server/configure-peering/configure-a-private-server.md [hunk 2]
```diff
@@ -40,9 +40,11 @@ To set up a specific server as a private peer, complete the following steps:
...
     If your server connects using **proxies**, the IP addresses and ports should match the configurations of the `rippled` servers you are using as proxies. For each of those servers, the port number should match the `protocol = peer` port in that server's config file (usually 51235). For example, your configuration might look like this:
 
-          [ips_fixed]
-          192.168.0.1 51235
-          192.168.0.2 51235
+        [ips_fixed]
+        192.168.0.1 51235
+        192.168.0.2 51235
+
+    **Note:** If you omit the port number, the server uses port 2459, the IANA-assigned port for the XRP Ledger protocol. [New in: rippled 1.6.0][]
```
Comment (ryangyoung) [inline content/tutorials/manage-the-rippled-server/configure-peering/configure-a-private-server.md]: Link to peer protocol? (Maybe it's already linked in the page.)

**Review by ryangyoung — APPROVED** [review]
Looks good. Left two suggestions where a link back to the peer protocol page might be helpful, but purely suggestions.

**Review by gregtatcam — APPROVED** [review]

**Comment by mDuo13** [comment]
All the broken links in the report are spurious:

- https://twitter.com/xrpsymbol/status/1006925937571713025 came back up
- the 1.6.0 tag is expected to be broken and will work when the version is released

**Pull request merged**